### PR TITLE
Update former-padawan.md

### PR DESCRIPTION
**Pull request by jetfire158**

### CODE REVIEW

File: content/theme/characters/archetypes/jedi/former-padawan.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
\ No newline at end of file
+layout: archetype
+attributes:
+  awareness: 10
+  coordination: 10
+  knowledge: 9
+  physique: 8
+  presence: 9
+  reflex: 10
+
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/jedi/former-padawan.md:13]: Attributes are 2 points over, should total 54

**Review by jetfire158 — COMMENTED** [review]
fixed

File: content/theme/characters/archetypes/jedi/former-padawan.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+    bargain: 1
+    persuade: 1
+    politics: 1
+    lead: 1
+  reflex:
+    acrobatics: 2
+    dodge: 2
+    leap: 2
+    melee: 3
+
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/jedi/former-padawan.md:43]: Skills should have 4 more points as well

**Review by jetfire158 — COMMENTED** [review]
done

**Review by jeffhuyghebaert — APPROVED** [review]

**Pull request merged**